radixdlt/src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,42 @@
+# Default logging setup for ledger
+
+rootLogger.level = debug
+rootLogger.appenderRef.stdout.ref = STDOUT
+rootLogger.appenderRef.file.ref = FILE
+# Uncomment the following for JSON logging, together with the appender
+#rootLogger.appenderRef.json.ref = JSON
+
+appender.console.type = Console
+appender.console.name = STDOUT
+appender.console.layout.type = PatternLayout
+appender.console.layout.pattern = %d{ISO8601} [%p/%c{1}/%t] (%F:%L) - %m%n
+
+appender.rolling.type = RollingFile
+appender.rolling.name = FILE
+appender.rolling.fileName = logs/radixdlt-core.log
+appender.rolling.filePattern = logs/radixdlt-core-%d{yyy-MM-dd}.log
+appender.rolling.layout.type = PatternLayout
+appender.rolling.layout.pattern = %d{ISO8601} [%p/%c{1}/%t] (%F:%L) - %m%n
+appender.rolling.policies.type = Policies
+appender.rolling.policies.time.type = TimeBasedTriggeringPolicy
+appender.rolling.policies.time.interval = 1
+appender.rolling.policies.time.modulate = true
+appender.rolling.strategy.type = DefaultRolloverStrategy
+# Two weeks = 14 days of log rollover
+appender.rolling.strategy.max = 14
+
+#appender.json.type = RollingFile
```

Comment:
What's with all this commented out configuration stuff?

Reply:
It's there in order to make it easy to include if you want it.  Not going to turn it on by default as it consumes a lot of I/O, and CPU to generate and store.